tests/PHPStan/Analyser/data/strtotime-return-type-extensions.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace ClosureReturnTypeExtensionsNamespace;
+
+use function PHPStan\Testing\assertType;
+
+$strtotimeNow = strtotime('2022-03-03 12:00:00 UTC');
+assertType('int<1646308801, max>', $strtotimeNow);
+
+$strtotimeInvalid = strtotime('4 qm');
+assertType('false', $strtotimeInvalid);
+
+$strtotimeUnknown = strtotime(rand(0, 1) === 0 ? 'now': '4 qm');
+assertType('int|false', $strtotimeUnknown);
+
+$strtotimeUnknown2 = strtotime($undefined);
+assertType('(int|false)', $strtotimeUnknown2);
+
+$strtotimeCrash = strtotime();
+assertType('int|false', $strtotimeCrash);
+
+$strtotimeWithBase = strtotime('+2 days', time());
+assertType('int', $strtotimeWithBase);
```

Comment:
maybe 

```suggestion
assertType('positive-int', $strtotimeWithBase);
```

?

Reply:
Yes but that would require parsing the second argument properly, which I am not doing right now. Feel free to send a PR for that, it's beyond my skills here :D